GoogleDataTransport/GDTCORLibrary/GDTCORFlatFileStorage.m

```diff
@@ -507,6 +507,9 @@ + (NSString *)pathForTarget:(GDTCORTarget)target
                     qosTier:(NSNumber *)qosTier
              expirationDate:(NSDate *)expirationDate
                   mappingID:(NSString *)mappingID {
+  NSMutableCharacterSet *allowedChars = [[NSCharacterSet alphanumericCharacterSet] mutableCopy];
```

Comment:
nit: I think, it can be `[NSMutableCharacterSet alphanumericCharacterSet]`, but the concern is not big enough to restart CI again.

Reply:
Ack.